phoebusgen/__init__.py

```diff
@@ -103,12 +115,37 @@ def _update_font_def(file_path):
                     predefined_fonts[font.replace(' ', '')] = {'name': font, 'family': family, 'style': style, 'size': size}
         return predefined_fonts
 
+def _update_version_def(file_path): # modifies _version dict in place
+    if not _path.isfile(file_path):
+        print('File at this path does not exist: {}'.format(file_path))
```

Comment:
this should always work but maybe we should add a return statement here so we can't hit the `with open(file_path ... ` code?